File: Data Structures & Algorithms/subarray-sums-divisible-by-k/submission-0.java
class Solution {
    public int subarraysDivByK(int[] nums, int k) {
        int count= 0;
        int[] sum= new int[nums.length];
        sum[0]= nums[0];
        for(int i= 1; i<nums.length; i++){
            sum[i]= sum[i-1]+nums[i];
        }

        for(int i= 0; i<nums.length; i++){
            if(sum[i]%k==0){
                count++;
            }

            for(int j= 0; j<i; j++){
                if((sum[i]-sum[j])%k==0){
                    count++;
                }
            }
        }
        return count;
    }
}

File: Data Structures & Algorithms/move-zeroes/submission-0.cpp
class Solution {
public:
    void moveZeroes(vector<int>& nums) {
        //brute force
        vector<int> temp;
        int count= 0;
        for(int i= 0; i<nums.size(); i++){
            if(nums[i]==0){
                count++;
            }else{
                temp.push_back(nums[i]);
            }
        }
        for(int i= 0; i<count; i++){
            temp.push_back(0);
        }
        nums= temp;
    }
};